runner.go

```diff
@@ -120,32 +122,41 @@ func makeExecutorReturnVals(funcType reflect.Type) func(sql.Result, error) []ref
 	if numOut == 1 {
 		return func(result sql.Result, err error) []reflect.Value {
 			if err != nil {
-				return []reflect.Value{zero}
+				return []reflect.Value{zeroInt64}
```

Comment:
Need to handle returning either a zero value int64 or a zero value sql.Result, depending on sType.